subsys/mpsl/fem/mpsl_fem_host.c

```diff
@@ -86,15 +86,9 @@ static int fem_nrf21540_gpio_pins_forward(void)
 
 #if DT_NODE_HAS_STATUS(MPSL_FEM_SPI_IF, okay)
 	uint8_t cs_pin = DT_SPI_DEV_CS_GPIOS_PIN(MPSL_FEM_SPI_IF);
-	uint8_t sck_pin = DT_PROP(DT_BUS(DT_NODELABEL(nrf_radio_fem_spi)), sck_pin);
```

Comment:
Please add a comment here that the reason for only the CS pin being forwarded is that the other pins may be used by the application core to communicate with other SPI peripherals.

Reply:
Done